src/chainstate/burn/db/sortdb.rs

```diff
@@ -699,6 +731,23 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
 // update this to add new indexes
 const LAST_SORTITION_DB_INDEX: &'static str = "index_burn_header_hash_pox_valid";
 
+const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[r#"
+    CREATE TABLE peg_in (
+        txid TEXT NOT NULL,
+        vtxindex INTEGER NOT NULL,
+        block_height INTEGER NOT NULL,
+        burn_header_hash TEXT NOT NULL,
+        sortition_id TEXT NOT NULL,
+
+        recipient TEXT NOT NULL,            -- Stacks principal to receive the sBTC, can also be a contract principal
+        peg_wallet_address TEXT NOT NULL,
+        amount TEXT NOT NULL,
+        memo TEXT,
+
+        PRIMARY KEY(txid, sortition_id)
```

Comment:
Hey, I know I said that this needs to be `(txid, sortition_id)` earlier, but it turns out I misspoke.  It should be `(txid, burn_header_hash)`.  It turns out that my PR #3528 is wrong as well.

Reply:
I left an explanation as to why #3528 is currently mis-designed here: https://github.com/stacks-network/stacks-blockchain/pull/3528#issuecomment-1408859705

Reply:
Alright, thanks for letting me know. I've updated the PR now using `(txid, burn_header_hash)` as the new primary key.